gem/models.py

```diff
@@ -15,6 +17,28 @@
 from wagtail.images.edit_handlers import ImageChooserPanel
 
 
+class VideosPage(Page):
```

Comment:
If this were to inherit from ArticlePage (molo.core.ArticlePage), then all these field would already be here. This model doesn't offer any functionality that the ArticlePage doesn't already offer.